distribution/alliance/pom.xml

```diff
@@ -395,58 +395,41 @@ org.codice.alliance.corba_default_port=2809</concat>
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack-alliance-docs</id>
+                        <!--
+                        The unpack needs to run in an earlier phase than the copy,
+                        due to a recurring bug in maven that causes plugins in the
+                        same lifecycle phase to run in orders other than that which
+                        they were declared in.
+
+                        See https://issues.apache.org/jira/browse/MNG-5799 as well
+                        as the related tickets for more details.
+                        -->
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
```

Comment:
❓ Do we need to add a dependency on the `org.codice.alliance:docs` artifact in the `<dependencies>` section?